docs/installation/pipx-based.rst

```diff
@@ -197,7 +224,9 @@ In both cases, if you run ``python --version`` afterwards, you should get ``Pyth
 Installing ``pipx``
 ~~~~~~~~~~~~~~~~~~~
 
-Next we need to install ``pipx``, a Python application manager that facilitates installing, keeping applications updated and run them in their own isolated environments. We could skip this step and install MUSE directly, but that will risk to have conflicting dependencies in the future if you install any other application, breaking your MUSE installation, and we do not want that to happen.
+Next we need to install ``pipx``, a Python application manager that facilitates installing, keeping applications updated and run them in their own isolated environments.
+More specifically, ``pipx`` will create a virtual environment to run the tools it installs based on the python version that was used to install pipx to start with, unless you specify another version and that other version is system wide available.
+We could skip this step and install MUSE directly, but that will risk to have conflicting dependencies in the future if you install any other application, breaking your MUSE installation, and we do not want that to happen.
```

Comment:
I'm not sure this is quite right. You can point `pipx` at a different Python executable and it'll use it, but it doesn't have to be available systemwide and the way it's written makes it sound like you can just tell it what version of Python to use -- which would be cool, but it doesn't do that.

I'd just drop these two sentences. I think the second one is confusing too.

Reply:
No @alexdewar, @HarmonicReflux is correct. You can tell `pipx` which python version to use when installing a package... as long as that python version is already available. So, if you do: `pipx install muse_os --python=/bin/python3.10`, it will install `muse` in a python 3.10 environment as long as you indicated the path to the existing python3.10 executable, regardless of how `pipx` was installed.

Reply:
If you have `pipx` installed, run `pipx install -h` and check the options.

Reply:
Right, but that `python3.10` *doesn't* have to be available systemwide! It can be installed wherever (e.g. `~/my_programs/my_python`).

I more meant that you have to tell `pipx` the path to a Python executable rather than just saying "please install using Python 3.7", but I didn't word it particularly well :upside_down_face: -- I know that's not what @HarmonicReflux wrote but I just thought it was a bit ambiguous. But anyway, if users have installed `pipx` via `pyenv` it'll already be using right version of Python, right? So I don't think we want to confuse things by telling users about this option.

Sorry to be pedantic. It's not the most important thing and it's ok as is. I don't want to drag this out for the sake of it...